### DISCUSSION TITLE
Curhat dikit bang(banyak si wkwk)🙏🙏

### DISCUSSION THREAD

**Discussion by AhriDev** [deaafrizal/forum#82] [The Lounge]
Jadi begini, Bang. Saya memang sudah memiliki passion di dunia teknologi (programming) sejak SMP. Saat SMP, saya sudah mempelajari fundamental HTML dan CSS. Ketika SMK, saya memilih jurusan RPL, tetapi di awal-awal SMK, saya mengalami beberapa masalah hingga akhirnya menjual laptop dan hampir meninggalkan dunia programming.

Saat PKL, saya kebetulan ditempatkan di instansi pendidikan Vokasi di Cimahi. Di sana, minat saya terhadap dunia programming kembali muncul karena saya diberi tugas untuk mempelajari visual scripting di Unreal Engine 5 untuk membuat game sederhana. Alhamdulillah, kinerja saya selama PKL dinilai memuaskan oleh pembimbing. Bahkan, setelah PKL selesai, saya masih sering diminta buat bantu-bantu dan ditawari pekerjaan setelah lulus.

Setelah lulus, saya diberi laptop bekas oleh pembimbing karena saya tidak memiliki laptop. Saya pun disuruh untuk mempelajari kembali UE5, Laravel, dan Unity. Sebulan setelah lulus, saya dipanggil untuk membantu mengerjakan proyek yang sedang dibuat oleh pembimbing. Karena saya tahu mencari pekerjaan saat ini sulit, saya pun menerima tawarannya.

Masalahnya, saya mengira pekerjaan itu sesuai dengan materi yang saya pelajari dan saya pikir saya akan bekerja di sana dalam waktu lama. Ternyata, jobdesknya berbeda dengan materi yang saya pelajari, dan mungkin karena performa saya yang kurang baik saat itu, akhirnya saya hanya dikontrak selama sebulan.

Dari situ, saya merasa down dan bingung. Apakah saya harus melanjutkan di bidang IT atau beralih ke bidang lain? Hingga akhirnya, saya menganggur selama setahun ini. Sampai saat ini, saya merasa putus asa dan bingung, Bang. Apakah saya harus melanjutkan di bidang IT atau berhenti saja? Karena jika saya melanjutkan di bidang IT, bagaimana pathnya? Saya hanya lulusan SMK, dan ilmu yang saya dapatkan juga nanggung-nanggung karena alasan di atas.

inimah tolong banget pencerahannya bang🙇‍♂️🙏🙏

**Comment by AhriDev** [comment]
barangkali ada yang mau jadi mentor saya hehe

**Reply by Dlanang** [reply]
semangat bang, terus berjuang dan jangan menyerah. karena saya juga awam dan baru mulai jadi belum bisa memberi saran.

**Reply by imamsudarajat04** [reply]
semangat bang, yang penting dalemin aja beberapa bahasa pemograman, nanti juga bakal ketemu kerjaan yang jobdesknya cocok :)

**Reply by AhriDev** [reply]
Makasih semuanya.
Btw, ada yang ikut program google cloud arcade facilitator?
